docs/gatsby-node.ts

```diff
@@ -252,6 +252,75 @@ export const createSchemaCustomization: GatsbyNode["createSchemaCustomization"]
 }) => {
   const typeDefs = [
     `
+    type Contributor implements Node @dontInfer {
```

Comment:
To be honest, I haven't done further research about changes in Gatsby in `docs`, as this project was not updated for a long time and now it requires advanced maintenance.